### ISSUE TITLE
Error when compiling from source

### ISSUE THREAD

**Issue by TheBearodactyl** [wbcolon/xPlay#1]
**Describe the bug**
Build error

**To Reproduce**
1. ```mkdir build && cd build```
2. ```cmake -S .. -B .```
3. ```make```
4. 
```make
[  2%] Automatic MOC and UIC for target xPlay
[  2%] Built target xPlay_autogen
[  4%] Automatic RCC for xPlay.qrc
[  6%] Building CXX object CMakeFiles/xPlay.dir/xPlay_autogen/mocs_compilation.cpp.o
In file included from /home/emi/.junk-drawer/xPlay/build/xPlay_autogen/EWIEGA46WW/moc_xPlayerConfiguration.cpp:10,
                 from /home/emi/.junk-drawer/xPlay/build/xPlay_autogen/mocs_compilation.cpp:1:
/home/emi/.junk-drawer/xPlay/build/xPlay_autogen/EWIEGA46WW/../../../xPlayerConfiguration.h:619:37: error: ‘XPLAY_APP_NAME’ was not declared in this scope
  619 |     const QString ApplicationName = XPLAY_APP_NAME;
      |                                     ^~~~~~~~~~~~~~
make[2]: *** [CMakeFiles/xPlay.dir/build.make:100: CMakeFiles/xPlay.dir/xPlay_autogen/mocs_compilation.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:84: CMakeFiles/xPlay.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
```

**Expected behavior**

A successful build

**Comment by wbcolon** [comment]
You currently need to add the release type using  -DCMAKE_BUILD_TYPE=RELEASE or  -DCMAKE_BUILD_TYPE=DEBUG
An install target is also currently missing, but you only need the xPlay binary. 
I will work on fixing these issues.

**Comment by TheBearodactyl** [comment]
> You currently need to add the release type using -DCMAKE_BUILD_TYPE=RELEASE or -DCMAKE_BUILD_TYPE=DEBUG An install target is also currently missing, but you only need the xPlay binary. I will work on fixing these issues.

Oki!

**Comment by wbcolon** [comment]
Should be fixed by commit d1e2507. Defaults to RELEASE built type if not specified. Also added simple install target.

**Comment by TheBearodactyl** [comment]
> Should be fixed by commit [d1e2507](https://github.com/wbcolon/xPlay/commit/d1e25070a286bc952a9c7661178a02a782dc7e15). Defaults to RELEASE built type if not specified. Also added simple install target.

thanks!
